src/tsf/TfDispAttr.cpp

```diff
@@ -54,7 +51,7 @@ HRESULT CicDisplayAttributeMgr::GetDisplayAttributeTrackPropertyRange(TfEditCook
     HRESULT hr = E_FAIL;
 
     ULONG ulNumProp;
-    ulNumProp = (ULONG) m_DispAttrProp.Count();
+    ulNumProp = (ULONG) m_DispAttrProp.size();

```

Comment:
can you change the c-style cast to a `static_cast` or if necessary a `reinterpret_cast`? We try to use the least powerful cast possible to effect the change we need.